controller/instance_manager_controller.go

```diff
@@ -940,14 +940,14 @@ func formatInstanceMessage(im *longhorn.InstanceManager) string {
 		}
 	}
 
-	if len(im.Status.Instances) > 0 {
+	if len(im.Status.Instances) > 0 { // nolint: staticcheck
```

Comment:
Any idea why we need to have `nolint` here?

Reply:
Oh. ok they are deprecated fields as @james-munson pointed out here https://github.com/longhorn/longhorn-manager/pull/3069